pkg/resources/common/common_test.go

```diff
@@ -1,7 +1,18 @@
 package common
 
 import (
+	"fmt"
 	"testing"
+
+	"github.com/rancher/webhook/pkg/admission"
+	"github.com/stretchr/testify/require"
+	authorizationv1 "k8s.io/api/authorization/v1"
+	v1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apiserver/pkg/authentication/user"
+	k8fake "k8s.io/client-go/kubernetes/typed/authorization/v1/fake"
+	k8testing "k8s.io/client-go/testing"
 )
 
 func TestIsModifyingLabel(t *testing.T) {
```

Comment:
I think there's one test missing from this - the case where a the labels exist and are populated but there's no change at all. This causes a line to be missing from the coverage stats of the test, would you mind adding that?